accounting_reporting_core/build.gradle.kts

```diff
@@ -5,6 +5,8 @@ dependencies {
     implementation("org.springframework.modulith:spring-modulith-api")
     implementation("org.springframework.modulith:spring-modulith-events-api")
 
+    implementation("org.springframework.boot:spring-boot-starter-security:3.2.4")
```

Comment:
Perhaps specifying version is not even needed as the one from spring boot will be taken automatically?